x11-wm/xmonad/Manifest

```diff
@@ -1 +1,2 @@
 DIST xmonad-0.17.0.tar.gz 93489 BLAKE2B 6990bc408194c99378f949b28f2115b7096e6474b5305f935be2c8af1eba4bff3e3c05477af9f833d1b52422e4fb95f857398b05f7f1081e3a2cc5dd8c50fc3a SHA512 1ec46ee61f4bf1879ea638e9be83378273050670ed2ee0f00e7d43b188cc29a874aa5b40076162a1ef0038802cc43f675773951f9b7d1f26329f9d105c28b3f7
+DIST xmonad-0.17.1.tar.gz 94704 BLAKE2B ea5d5845ded40449916b0b213766b08f47e029c34ef87eeed27c3591cfcc8efe202db59e6e83373d48626860fa0248b0ac46aaf8c7d2725c1c994463e7478b6e SHA512 cf8aa854112f91298936386681b474daa33ed782459061a0542679359abd06edaa9748a66e2b9f6b9f75eb0078bc728c0d565fcf758105a28c4ba6dda6a4959a
```

Comment:
Please also remove `xmonad-0.17.0.ebuild`, update the manifest (`pkgdev manifest`), and amend/fixup/squash the changes to cf0e545eded6ea310139dca8581e1799778ead6a.

This will

a. Remove the old version
b. Trigger a diff view in github between the two versions

Reply:
Got it. I've removed `xmonad-0.17.0.ebuild`.